astropy/io/ascii/latex.py

```diff
@@ -220,17 +220,28 @@ class Latex(core.BaseReader):
 
     Examples::
 
-        caption = r'My table \label{mytable}'
-        caption = 'My table \\\\label{mytable}'
+        >>> caption = r'My table \label{mytable}'
+        >>> caption = 'My table \\\\label{mytable}'
 
     **latexdict** : Dictionary of extra parameters for the LaTeX output
 
         * tabletype : used for first and last line of table.
             The default is ``\\begin{table}``.  The following would generate a table,
             which spans the whole page in a two-column document::
 
-                ascii.write(data, sys.stdout, Writer = ascii.Latex,
-                            latexdict = {'tabletype': 'table*'})
+              >>> from astropy.io import ascii
+              >>> import sys
+              >>> data = {'name': ['bike', 'car'], 'mass': [75,1200], 'speed': [10, 130]}
```

Comment:
This is an astronomy package, so if you must include data in the example, please make it astronomy focused, or purely generic such as "cola" and "colb".

Reply:
Do we need to turn every line of code into a doctest? We do have the narrative docs for io.ascii. I see that point that doctests make sure that these examples never go out of date, but it also makes for very long docstrings. That's fine in the rendered webpages, but not as comfortable when I to `help( ...)` on the command line or `( + TAB` in a notebook. I'm not sure that having the full LaTeX output for all example is worth it - but then again it might help people so see what exactly the output looks like.

Reply:
I agree with @hamogu that we do not need to have every code snippet in code docstrings be a running doctest. IMO the added code output here *reduces* the clarity of the documentation from the user perspective. I would suggest not including the output and skipping the test. Basically this kind of information about what the output looks like belongs only in the narrative docs.